wazo_acceptance/environment.py

```diff
@@ -53,7 +52,9 @@ def initialize(context):
     config = load_config(config_dir=context.config.userdata.get('acceptance_config_dir'))
     wazo_setup_logging(config['default']['log_file'], foreground=True, debug=config['default']['debug']['global'])
```

Comment:
This causes issues when the default section is removed

Reply:
fixed